terraform/modules/bootstrap_concourse/sg_bootstrap.tf

```diff
@@ -0,0 +1,22 @@
+resource "aws_security_group" "bootstrap" {
+  ingress {
+    from_port = 22
+    to_port = 22
```

Comment:
I added this as a debugging option since this thing creates the world, if something goes wrong you might want to be able to find out why, especially if something goes wrong before concourse runs your job and you have nothing to hijack.

However I haven't actually needed this yet in my testing, so I can remove it if you feel strongly about it.